audformat/core/testing.py

```diff
@@ -392,4 +395,10 @@ def to_scheme_type(series: pd.Series) -> str:
     db['misc']['gender'] = Column(scheme_id='gender')
     db['misc']['gender'].set(db.schemes['gender'].draw(len(index)))
 
+    ############################
+    # Schemes from Misc Tables #
+    ############################
+
+    db.schemes['speaker'] = Scheme(labels='misc', dtype='str')
```

Comment:
I think we should add a column to `db['files']` and `db['segments']` where we actually define this scheme.